src/database/models/organization_item.rs

```diff
@@ -0,0 +1,361 @@
+use std::collections::HashMap;
+
+use crate::models::{
+    ids::base62_impl::{parse_base62, to_base62},
+    teams::ProjectPermissions,
+};
+
+use super::{ids::*, TeamMember};
+use redis::cmd;
+use serde::{Deserialize, Serialize};
+
+const ORGANIZATIONS_NAMESPACE: &str = "organizations";
+const ORGANIZATIONS_SLUGS_NAMESPACE: &str = "organizations_slugs";
+
+const DEFAULT_EXPIRY: i64 = 1800;
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+/// An organization of users who together control one or more projects and organizations.
+pub struct Organization {
+    /// The id of the organization
+    pub id: OrganizationId,
+
+    /// The slug of the organization
+    pub slug: String,
+
+    /// The associated team of the organization
+    pub team_id: TeamId,
+
+    /// The name of the organization
+    pub name: String,
+
+    /// The description of the organization
+    pub description: String,
+
+    /// Default project permissions for associated projects
+    pub default_project_permissions: ProjectPermissions,
+
+    /// Any associated urls for the organization
+    pub urls: HashMap<String, String>,
```

Comment:
i think this should be an enum with possible allowed links

Reply:
might want to talk on vc on how we structure this

Reply:
I set it up like donation links- we can discuss monday if you want it a specific different way